packages/react/src/components/breadcrumbs/index.ts

```diff
@@ -3,11 +3,56 @@ import { BreadcrumbsItem } from './breadcrumbs-item';
 import { BreadcrumbsLink } from './breadcrumbs-link';
 import { BreadcrumbsList } from './breadcrumbs-list';
 
+type BreadcrumbsCompoundProps = typeof BreadcrumbsParent & {
```

Comment:
Forgot to mention, but suggest we all these the same name as "parent" component. 

The type is show when hovering over the component.

<img width="418" height="85" alt="Image" src="https://github.com/user-attachments/assets/d1c5fbae-a8cc-4b7a-b980-6667bfbcea55" />

Reply:
I think I'll just rename the const and then `export {CardComponent as Card }` etc